repose-aggregator/src/docs/asciidoc/services/datastores.adoc

```diff
@@ -1,9 +1,5 @@
 = Datastores
 
-include::../_includes/in-progress.adoc[]
-
-== Configuration
-
```

Comment:
Sidenote: You can't send a link with line numbers to a `.adoc` file. Or at least one that is meaningful.
> https://github.com/rackerlabs/repose/blob/13ea624326755a8333cbff00b8f20369454fe174/repose-aggregator/src/docs/asciidoc/services/datastores.adoc#L3-L10
means absolutely nothing.

However a `blame` view does.
> https://github.com/rackerlabs/repose/blame/13ea624326755a8333cbff00b8f20369454fe174/repose-aggregator/src/docs/asciidoc/services/datastores.adoc#L3-L10

Ok, so there is a start to a good opening, but I agree it could use some beefing up.